pom.xml

```diff
@@ -5,8 +5,9 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.example</groupId>
-    <artifactId>My-own-project2-Cinema</artifactId>
+    <artifactId>String.inrto</artifactId>
```

Comment:
spring? :)

Reply:
I get the same issue with "sprint" in my case :D

Reply:
:)))Done